lib/src/layer/tile_layer/tile_image_view.dart

```diff
@@ -3,6 +3,7 @@ import 'dart:collection';
 import 'package:flutter_map/flutter_map.dart';
 import 'package:flutter_map/src/layer/tile_layer/tile_range.dart';
 
+/// A
```

Comment:
Needs finishing.

Reply:
woops, that's an oversight. Thanks for bringing this up 😄 fixed